cassdegrees/templates/managecourses.html

```diff
@@ -0,0 +1,115 @@
+{% extends "base.html" %}
+
+{% block page-title %}Manage Courses{% endblock %}
+
+{% block subtitle %}Add, Modify or Delete Courses{% endblock %}
+
+{% block content %}
+    <form class="anuform" action="/manage_courses/" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <legend>Add Course</legend>
+
+            <!-- As HTML forms only support get or post methods,
+            add this hidden input for consistency with the rest of the forms -->
+            <input type="hidden" name="_method" value="post">
+
+            <p>
+                <label for="id">Course Name:</label>
+                <input class="text" id="text" type="text" name="name" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="id">Course Code:</label>
+                <input class="text" id="text" type="text" name="code" minlength="8" maxlength="8" size="8" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="text">Year:</label>
+                <!-- static min max values are only temporary -->
+                <input class="text" id="number" type="number" name="year" min="1990" max="2100" aria-required="true" required>
```

Comment:
```suggestion
                <input class="text tfull" id="number" type="number" name="year" min="1990" aria-required="true" required>
```

Changes will make the form stretch across to the end of the page. Also remove max year, it isn't needed.

Reply:
my thought process was so it leaves the user less of a chance to create a mistake when creating a course, hence adding max for both year. My original intention was to make it dynamic but will do that at a later date once the main work has been approved and if everyone is ok with this.

Reply:
I still don't think we should restrict the software like that though. Maybe we could ask the user for confirmation if the entered year is too distant from the current year (Implementation may be too complicated for the resulting gain though)